### PR TITLE
Require Alias and Context for *-APP commands

### PR DESCRIPTION
**Pull request by jajik**
https://issues.redhat.com/browse/MODCLUSTER-832

### CODE REVIEW

**Comment by jajik** [comment]
Converting to draft because it does make sense to have some APP commands global... have to sort it out...

File: native/mod_manager/mod_manager.c [hunk 1]
```diff
@@ -2107,6 +2108,13 @@ static char *process_appl_cmd(request_rec *r, char **ptr, int status, int *errty
         *errtype = TYPESYNTAX;
         return SROUBAD;
     }
+
+    /* Note: This applies only for non-wildcared requests for which Alias and Context are required */
```
Comment (rhusar) [inline native/mod_manager/mod_manager.c:2112]: What is 'wildcared'? :)

**Review by rhusar — CHANGES_REQUESTED** [review]

File: native/mod_manager/mod_manager.c (hunk 1 above)
Comment (jajik) [reply native/mod_manager/mod_manager.c]: Good catch, should be ok now.

Comment (rhusar) [reply native/mod_manager/mod_manager.c]: Haha, thanks

**Comment by rhusar** [comment]
@jajik Can you remind me what are the implications of mod_proxy_cluster creating an empty Virtual Host (with empty Context and Alias)?

**Review by jfclere — APPROVED** [review]

**Comment by jajik** [comment]
@rhusar could you please remove your *changes requested* if there is nothing else?

**Comment by rhusar** [comment]
Thanks @jajik and @jfclere

**Pull request merged**